### PR TITLE
Extend SinkBase to support DOC-file for frontend ragnarok

### PR DESCRIPTION
**Pull request by jbtfk**
I have tested this by adding SINK.md to the root of a Sink implementation, called `$this->getDocumentation()` from somewhere in Sink (like fetch). Then I've printed the result to log by using LogPrintf.

### CODE REVIEW

File: SINK.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Sink (Sink Title?)
+Short description of sink
```
Comment (tfk-kaare) [inline SINK.md:3]: Add some space around headers/content

File: src/Sinks/SinkBase.php [hunk 2]
```diff
@@ -4,6 +4,8 @@
 
 use Illuminate\Support\Carbon;
 use Ragnarok\Sink\Models\SinkFile;
+use League\CommonMark\GithubFlavoredMarkdownConverter;
```
Comment (tfk-kaare) [inline src/Sinks/SinkBase.php:7]: composer.json dependency is probably required for this?

**Review by tfk-kaare — COMMENTED** [review]
Beautiful!

Documentation nitpic, and question regarding the source of github flavored markdown converter. Where does it come from? :-)

Otherwise I love the simplicity and clean solution on this!

File: src/Sinks/SinkBase.php (hunk 2 above)
Comment (jbtfk) [reply src/Sinks/SinkBase.php:7]: CommonMark is a dependency of Laravel Framework. Might be needed in ragnarok-sink for isolated unit-testing?
As long as ragnarok-sink runs as a laravel package in a laravel installation League\CommonkMark\GithubFlavoredMarkdownConverter is available. 

I'm no expert in Laravel though, so maybe there should be a reference to league/commonmark in composer.json for ragnarok-sink?

File: SINK.md (hunk 1 above)
Comment (jbtfk) [reply SINK.md:3]: Fixed

**Review by tfk-kaare — APPROVED** [review]

**Pull request merged**